public/javascripts/SVLabel/src/SVLabel/mission/Mission.js

```diff
@@ -153,6 +153,10 @@ function Mission(parameters) {
             };
             svl.labelCounter.reset();
         }
+
+        if (svl.main.isAnAnonymousUser() && !svl.onboarding.isOnboarding()){
```

Comment:
This line creates an error:

![image](https://user-images.githubusercontent.com/2873216/27920305-1ffbc8f2-6242-11e7-8a43-21bf4b0f200b.png)